src/server/hotspot_partition_calculator.h

```diff
@@ -40,14 +47,14 @@ class hotspot_partition_calculator
 
 private:
     const std::string _app_name;
-
     void init_perf_counter(int perf_counter_count);
     // usually a partition with "hot-point value" >= 3 can be considered as a hotspot partition.
```

Comment:
I don't understand this comment, could you please expain it to me?

Reply:
"hot-point value" is a value of partition to show whether it is hot. It is an int type usually ranges from 0  to 10, if one partition's value >=3, it can be considered as a hotspot partition.